client/Router.jsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
+// import axios from 'axios';
+//import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { BrowserRouter as BrowserRouter, Switch, Route } from 'react-router-dom';
+import { AuthProvider } from './AuthContext';
+import { Container, CssBaseline } from '@material-ui/core';
+import {SignUp} from './screens/Signup';
+import {UseStyles} from './components/UseStyles';
```

Comment:
Describe a bit in discord, nit: move `useStyles` to `/theme` or `/styles` or something outside `/components`

Reply:
![image](https://user-images.githubusercontent.com/35232697/105070975-9f7fd700-5a49-11eb-862a-12aff829abcc.png)